pyteal/compiler/compiler.py

```diff
@@ -306,6 +306,7 @@ def __init__(
         *,
         version: int = DEFAULT_PROGRAM_VERSION,
         assemble_constants: bool = False,
+        assembly_type_track: bool = True,
```

Comment:
Also added the option to include `#pragma typetrack false`. This was enabled for integration tests briefly due to the assembler type error, but the latest docker image has that fix included, so it's no longer needed.

Still, I though it'd be useful to include this.